### PR TITLE
[Docs] fin documentation

### PR DESCRIPTION
**Pull request by luarss**
Added License and missing params

### CODE REVIEW

File: src/fin/README.md [hunk 1]
```diff
@@ -58,6 +72,12 @@ parser accepts but ignores (e.g., `space_to_outline`).
 
 ## Regression tests
 
+There is a set of regression tests in `/test`.
```
Comment (dralabeing) [inline src/fin/README.md:75]: Please include note on regression tests as discussed for all tools.

**Comment by luarss** [comment]
@dralabeing Added the link.

**Review by vvbandeira — APPROVED** [review]
@luarss I see that when I click on the regression link, the page rolls, and the header ends up at the bottom of the screen. Can we make it go to the top of the page?

File: src/fin/README.md [hunk 2]
```diff
@@ -58,6 +72,14 @@ parser accepts but ignores (e.g., `space_to_outline`).
 
 ## Regression tests
 
+There are a set of regression tests in `/test`. For more information, refer to this [section](../../README.md#regression-tests).
```
Comment (vvbandeira) [inline src/fin/README.md:75]:
```suggestion
There are a set of regression tests in `./test`. For more information, refer to this [section](../../README.md#regression-tests). 
```

**Comment by luarss** [comment]
@vvbandeira It's weird, because when I click on the right sidebar the anchor links rise correctly. I filed an issue with the sphinx-book-theme repo so we'll see

**Comment by vvbandeira** [comment]
> @vvbandeira It's weird, because when I click on the right sidebar the anchor links rise correctly. I filed an issue with the sphinx-book-theme repo so we'll see

I tried in two different browsers and noticed the following. Looks to be an issue where the page goes to the correct place, but then the images finish loading and the page scrolls down.

On a Chrome-based web-browser:

https://github.com/The-OpenROAD-Project/OpenROAD/assets/9001905/c6cdf60d-88ab-41e7-ae66-2e9e7ece8a29

On Safari:

https://github.com/The-OpenROAD-Project/OpenROAD/assets/9001905/f401f824-525d-4785-bd4a-be1d65b2a9bd

File: src/fin/README.md [hunk 3]
```diff
@@ -6,11 +6,25 @@ configuration file.
...
+    [-area {lx ly ux uy}]
 ```
 
-If `-area` is not specified, the core area will be used.
+#### Options
+
+| Switch Name | Description | 
+| ----- | ----- |
+| `-rules` | specify json rule file |
+| `-area` | optional. If not specified, the core area will be used. |
```
Comment (vvbandeira) [inline src/fin/README.md:27]:
```suggestion
| `-rules` | Specify `json` rule file. |
| `-area` | Optional. If not specified, the core area will be used. |
```

**Pull request merged**